testing/testcafe/tests/form/form.buttonItem.ts

```diff
@@ -1,17 +1,18 @@
 import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
-import url from '../../../helpers/getPageUrl';
-import createWidget from '../../../helpers/createWidget';
+import url from '../../helpers/getPageUrl';
+import createWidget from '../../helpers/createWidget';
+import { testScreenshot } from '../../helpers/themeUtils';
 
-fixture`Form`
-  .page(url(__dirname, '../../container.html'));
+fixture.disablePageReloads`Form`
+  .page(url(__dirname, '../containerQuill.html'));
```

Comment:
why quill?
```suggestion
  .page(url(__dirname, '../container.html'));
```

Reply:
Form uses htmlEditor inside

Reply:
got it, HtmlEditor is used inside the form